data/tlg2042/tlg086/tlg2042.tlg086.1st1K-grc1.xml

```diff
@@ -0,0 +1,459 @@
+                <title xml:lang="lat">Homiliae in Job (fragmenta in catenis, typus I+II) (e codd. Vat.)</title>
+                <author>Origen</author>
+                <editor>Jean Baptiste Pitra</editor>
+                <funder>Center for Hellenic Studies</funder>
+                <respStmt>
+                    <resp>Published original versions of the electronic texts</resp>
+                    <orgName ref="https://www.opengreekandlatin.org">Open Greek and Latin</orgName>
+                    <persName role="principal">Gregory Crane</persName>
+                    <persName role="principal">Leonard Muellner</persName>
+                    <persName role="principal">Bruce Robertson</persName>
+                </respStmt>
+            <respStmt>
+                <persName>Charlotte Houghton</persName>
+                <orgName/>
+                <resp>Digital conversion and editing</resp>
+            </respStmt>
+                <respStmt>
+                    <persName>Domi Couvet</persName>
+                    <orgName>Center for Hellenic Studies</orgName>
+                    <resp>Digital conversion and editing</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Theodoros Pazarloglou</persName>
+                    <orgName>Center for Hellenic Studies</orgName>
+                    <resp>Digital conversion and editing</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Dora Profyri</persName>
+                    <orgName>Center for Hellenic Studies</orgName>
+                    <resp>Digital conversion and editing</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Zeyu Wen</persName>
+                    <orgName>Center for Hellenic Studies</orgName>
+                    <resp>Digital conversion and editing</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Lisa Cerrato</persName>
+                    <orgName>Perseus Digital Library</orgName>
+                    <resp>Digital editor</resp>
+                </respStmt>
+            </titleStmt>
+            
+            <publicationStmt>
+                <publisher>Open Greek and Latin</publisher>
+                <idno type="filename">tlg2042.tlg086.1st1K-grc1.xml</idno>
+                <pubPlace/>
+                <availability><licence target="https://creativecommons.org/licenses/by-sa/4.0/">Available under a Creative Commons Attribution-ShareAlike 4.0 International License</licence></availability>
+                <date when="2023-06-20"/>
+            </publicationStmt>
+            
+            <sourceDesc>
+                <biblStruct>
+                    <monogr>
+                        <author>Origen</author>
+                        <title xml:lang="lat">Analecta sacra spicilegio solesmensi parata</title>
+                        <editor><persName>Jean Baptiste Pitra</persName></editor>
```

Comment:
Again, here editor encoded two different ways.

Reply:
This is how the file was encoded when I opened it plus probably copy/paste. Will fix.